rllib/algorithms/sac/torch/sac_torch_learner.py

```diff
@@ -258,8 +256,6 @@ def compute_loss_for_module(
         # If a twin Q network should be used, add the critic loss of the twin Q network.
         if self.config.twin_q:
             critic_twin_loss = torch.mean(
-                # TODO (simon): Introduce priority weights when episode buffer is ready.
```

Comment:
awesome!

Reply:
FInally we have it and I forgot about these TODOs :D